modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/KaleoDesignerPortlet.java

```diff
@@ -361,21 +361,12 @@ private void _serveScriptLanguages(
 			ResourceRequest resourceRequest, ResourceResponse resourceResponse)
 		throws Exception {
 
-		Set<String> supportedScriptLanguages =
-			ScriptingUtil.getSupportedLanguages();
-
-		Stream<String> supportedScriptLanguagesStream =
-			supportedScriptLanguages.stream();
-
-		List<Object> sortedSupportedScriptLanguages =
-			supportedScriptLanguagesStream.sorted(
-			).collect(
-				Collectors.toList()
-			);
-
 		JSONArray jsonArray = _jsonFactory.createJSONArray();
 
-		for (Object supportedScriptLanguage : sortedSupportedScriptLanguages) {
+		for (Object supportedScriptLanguage :
```

Comment:
@LisztLiszt It's already a `List<String>`, so it's redundant to declare the element as `Object` and then `toString()`.

Reply:
@LisztLiszt This for-loop can be simplified replaced with `JSONUtil.toJSONArray(Collection, UnsafeFunction, Log)`.